siemens_plc_interface:
  # Topic for subscribing msgs from Master HFD.
  topic_hfd_command: /hfd/master_hfd/hfd_command
  
  # Host of Siemens PLC server.
  host: 192.168.58.20
  # Port of Siemens PLC server.
  port: 502
  # Rate of Siemens PLC server.
  rate: 50  # Hz
  # Whether reset the Siemens PLC client.
  reset_registers: True
  # Topic for subscribing msgs from Siemens PLC server.
  sub_topic: modbus_wrapper/output
  # Topic for publishing msgs to Siemens PLC server.
  pub_topic: modbus_wrapper/input
